charts/victoria-logs-single/values.yaml

```diff
@@ -150,35 +150,35 @@ server:
     #   cpu: 500m
     #   memory: 512Mi
 
-  # Indicates whether the Container is ready to service requests. If the readiness probe fails, the endpoints controller removes the Pod's IP address from the endpoints of all Services that match the Pod. The default state of readiness before the initial delay is Failure. If a Container does not provide a readiness probe, the default state is Success.
-  readinessProbe:
-    httpGet:
-      path: /health
-      port: http
-    initialDelaySeconds: 5
-    periodSeconds: 15
-    timeoutSeconds: 5
-    failureThreshold: 3
-
-  # Indicates whether the Container is running. If the liveness probe fails, the kubelet kills the Container, and the Container is subjected to its restart policy. If a Container does not provide a liveness probe, the default state is Success.
-  livenessProbe:
-    httpGet:
-      scheme: HTTP
-      path: /health
-      port: 9428
-    initialDelaySeconds: 30
-    periodSeconds: 30
-    timeoutSeconds: 5
-    failureThreshold: 10
-
-  # Indicates whether the Container is done with potentially costly initialization. If set it is executed first. If it fails Container is restarted. If it succeeds liveness and readiness probes takes over.
-  startupProbe: {}
-    #tcpSocket:
-    #  port: http
-    #failureThreshold: 30
-    #periodSeconds: 15
-    #successThreshold: 1
-    #timeoutSeconds: 5
+  probe:
```

Comment:
Can we not adding extra `probe:` but using the old `readinessProbe`, `livenessProbe` fields?
So this doesn't become a breaking change.

Reply:
want to make it consistent across all charts. I don't care if it should be `probe.readiness` or `readinessProbe`, but I would like to have same option across all charts, instead of these two, that we're having at the moment

Reply:
okay, thanks